src/pages/data_submission/DataSubmissionForm.jsx

```diff
@@ -94,12 +94,29 @@ export const DataSubmissionForm = (props) => {
   // compute multipart/form-data object, includes registration information and all files
   const createMultiPartFormData = (registration) => {
     const multiPartFormData = new FormData()
-
     multiPartFormData.append('dataset', JSON.stringify(registration))
-
     for (const field of Object.keys(formFiles)) {
       if (!isNil(formFiles[field])) {
-        multiPartFormData.append(field, formFiles[field])
+        const files = formFiles[field]
+        if (Array.isArray(files)) {
+          files.forEach((file, idx) => {
+            const duosFileType = Object.keys(file)[0]
+            const duosFileObj = Object.values(file)[0]
```

Comment:
Nit suggestion
```suggestion
            const [duosFileType, duosFileObj] = Object.entries(file)[0]
```

Or this if we know there is exactly one key/value pair

```suggestion
  const [[duosFileType, duosFileObj]] = Object.entries(file)
```

Reply:
Good second suggestion.  You can see how the code evolved into determining what I needed.  This will be good cleanup to that discovery.